### PR TITLE
feat: app - Dashboard Home Treatment Follow Up

### PR DESCRIPTION
**Pull request by LeBaku**
# Description

Create the treatment follw up card in the dashboard

### Click Up reference

[CU-86bxb9tvx](https://app.clickup.com/t/86bxb9tvx)

# Changes include

- [ ] Bugfix (non-breaking change that solves an issue)
- [X] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Checklist

- [X] I have assigned this PR to myself
- [X] I have added at least 1 reviewer
- [X] I have added the needed labels
- [X] I have linked this PR to a clickup task
- [X] I have tested this code
- [X] I have added / updated tests (unit / functionals / end-to-end / ...)
- [ ] I have updated the README and other relevant documents (guides...)
- [ ] I have added sufficient documentation both in code, as well as in the READMEs

### CODE REVIEW

**Comment by Lucas-COX** [comment]
Task linked: [CU-86bxb9tvx app-dashboard-home-treatmentFollowUp](https://app.clickup.com/t/86bxb9tvx)

File: edgar-app/src/components/dashboardPages/home/HomeTreatmentsCard.tsx [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+			spacing="16px"
+			w={{ base: '100%', md: 'auto' }}
+			bg="white"
+			borderRadius="16px"
+			p="16px"
+			border="2px"
+			borderColor="blue.200"
+		>
+			<Box w="100%">
+				<Link href="/dashboard/chat">
```
Comment (lucas-louis) [inline edgar-app/src/components/dashboardPages/home/HomeTreatmentsCard.tsx:43]: Bad url

File: edgar-app/src/components/dashboardPages/home/HomeTreatmentsCard.tsx [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+										textDecoration={
+											checkedItems.includes(treatment.name) ? 'line-through' : undefined
+										}
+									>
+										{treatment.number} x {treatment.name}
+									</Text>
+								</Checkbox>
+							))}
+						</VStack>
+					</VStack>
```
Comment (lucas-louis) [inline edgar-app/src/components/dashboardPages/home/HomeTreatmentsCard.tsx:138]: You need to handle the data in another way, the backend will send you a list of treatments checked and you need to get the medical info of the patient and for each treatment you need to group and to display the treatment in the good order

File: edgar-app/src/components/dashboardPages/home/HomeTreatmentsCard.tsx [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+					bg="orange.100"
+					p="8px 16px"
+					spacing="8px"
+				>
+					<HStack spacing="16px" w="100%">
+						<Icon as={WarningIcon} w="32px" h="32px" color="orange.600" />
+						<Text color="orange.600" size="boldMd">
+							Vous n’avez pas encore <br />
+							enregistré de suivi sur vos <br />
+							traitements
```
Comment (lucas-louis) [inline edgar-app/src/components/dashboardPages/home/HomeTreatmentsCard.tsx:155]: You don't need to use `<br />`

**Review by lucas-louis — CHANGES_REQUESTED** [review]
You need to connect the backend

**Review by lucas-louis — APPROVED** [review]
LGTM

**Pull request merged**